tests/test_sklearn_tfidf_vectorizer_converter.py

```diff
@@ -29,7 +37,7 @@ def test_model_tfidf_vectorizer11(self):
         vect = TfidfVectorizer(ngram_range=(1, 1), norm=None)
         vect.fit(corpus.ravel())
         model_onnx = convert_sklearn(vect, "TfidfVectorizer",
-                                     [("input", StringTensorType([1]))],
+                                     [("input", StringTensorType())],
```

Comment:
Don't we need to pass shape?

Reply:
Not always. My goal is to test batch predictions so I need to remove that information.